README.md

```diff
@@ -207,7 +207,7 @@ For more details, please refer to the
 Initialization is usually performed under the hood and can be even passed over,
 as far as handles are created using the `Loop::resource` member function.<br/>
 On the other side, handles keep themselves alive until one explicitly closes
-them. Because of that, memory usage will grow up if users simply forget about a
+them. Because of that, memory usage will grow if users simply forget about a
```

Comment:
Just curious. Why is _up_ wrong here?

Reply:
you could use "up", though that's usually for people, like "the children grow up". It's not wrong, just possibly confusing. I'm happy to change it back if you would prefer the original.

Reply:
No, don't worry, I was just curious. You don't need to change it back. :wink: